node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric_test.go

```diff
@@ -330,16 +420,26 @@ func TestGetDmsPath(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			mockCtrl := gomock.NewController(t)
 			defer mockCtrl.Finish()
+			volumIds := []string{volumeUuid, volumeNguid}
```

Comment:
```suggestion
			volumeIdVariations := []string{volumeUuid, volumeNguid}
```
please ensure the tests are aligned when renaming

Reply:
done, but I didn't find `volumeUids` in the tests

Reply:
the `volumeUids` are not in the tests (otherwise I wouldn't need to mention it in addition to the tests)

Reply:
done